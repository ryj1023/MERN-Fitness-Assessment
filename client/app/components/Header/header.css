.header{
	background-image: url('../../images/website-background.gif');
	height: 75px;
    width: 485px;
    margin-top: 2%;
    margin-bottom: 2%;
    margin-left: 28%;
    position: center;
    border-radius: 25px;
    border-style: solid;
    border-color: #FFFFFF;
}

.title{
	margin: 2%;
  	text-align: center;
  	color: white;
  	font-size: 47px;
}
