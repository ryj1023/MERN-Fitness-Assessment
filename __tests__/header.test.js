import React from 'react'
import Header from '../client/app/components/headers'
import { shallow } from 'enzyme'

it('expect to render header component', () => {
    expect(shallow(<Header />)).toMatchSnapshot()
})
