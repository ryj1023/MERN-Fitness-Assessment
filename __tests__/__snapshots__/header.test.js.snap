// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render header component 1`] = `
<div
  className="header"
>
  <h1
    className="title text-center"
  />
</div>
`;
